src/templates/forum_conversation/topic_detail.html

```diff
@@ -0,0 +1,204 @@
+                       title="{% trans "Edit" %}"><i class="fas fa-edit"></i>&nbsp;{% trans "Edit" %}</a>
+                  {% endif %}
+                  {% get_permission 'can_delete_post' post request.user as user_can_delete_post %}
+                  {% if user_can_delete_post %}
+                    <a href="{% url 'forum_conversation:post_delete' forum.slug forum.pk topic.slug topic.pk post.pk %}"
+                       class="btn btn-danger btn-sm"
+                       title="{% trans "Delete" %}"><i class="fas fa-times"></i></a>
+                  {% endif %}
+                </div>
+                {% spaceless %}
+                  <h4 class="m-0 subject">
+                    {{ post.subject }}
+                    &nbsp;<a href="{% url 'forum_conversation:topic' forum.slug forum.pk topic.slug topic.pk %}?post={{ post.pk }}#{{ post.pk }}">¶</a>
+                  </h4>
+                {% endspaceless %}
+                <p>
+                  <small class="text-muted">
+                    {% spaceless %}
+                      <i class="fas fa-clock"></i>&nbsp;
+                      {% if post.poster %}
+                        {% url 'forum_member:profile' post.poster_id as poster_url %}
+                        {% blocktrans trimmed with poster_url=poster_url username=post.poster|forum_member_display_name creation_date=post.created %}
+                          By: <a href="{{ poster_url }}">{{ username }}</a> on {{ creation_date }}
+                        {% endblocktrans %}
+                      {% else %}
+                        {% blocktrans trimmed with poster_username=post.username creation_date=post.created %}
+                          By: {{ poster_username }} on {{ creation_date }}
+                        {% endblocktrans %}
+                      {% endif %}
+                    {% endspaceless %}
+                  </small>
+                </p>
+                <div class="post-content">{{ post.content.rendered }}</div>
+                {% include "forum_conversation/forum_attachments/attachments_detail.html" %}
+                {% if post.enable_signature and post.poster.forum_profile.signature %}
+                  <div class="post-signature">{{ post.poster.forum_profile.signature.rendered }}</div>
+                {% endif %}
+                {# Vote buttons #}
+                <div class="vote-container d-flex p-2 gap-3 align-items-center"
+                     data-post-id="{{ post.id }}"
+                     data-user-rating="{{ post.user_rating }}"
+                     data-rating="{{ post.rating }}">
+                  <button class="like-button btn btn-primary btn-sm"
+                          type="submit"
+                          name="rate"
+                          value="like">
+                    <i class="fas fa-thumbs-up"></i> Like
+                  </button>
+                  <span class="like-count fas">{{ post.rating }}</span>
```

Comment:
# Color like button text
 As noted by @joshuamonroe mentioned [here](https://github.com/uchicago-cs/chigame/pull/341#pullrequestreview-1765656393), we should have some color indicator based the user's vote.

e.g. the user disliked:
![image](https://github.com/uchicago-cs/chigame/assets/55334507/704caae8-718d-4bc0-bf2e-edbe1c474e53)

the user liked:
![image](https://github.com/uchicago-cs/chigame/assets/55334507/eeb359f5-73fb-458c-a9a6-28a7e516d6e5)

The colors used here are the same colors from the buttons:

```
#3579F6 // blue
#CB444A // red
```

Reply:
I have instead just changed the color of the icon as opposed to the number because I figured it would be somewhat convulated for sake of people like me who are colorblind LOL. Attached are photos of the changes instead. 
<img width="313" alt="Screenshot 2023-12-05 at 2 55 09 PM" src="https://github.com/uchicago-cs/chigame/assets/121977204/d921f303-84f8-4e52-ac04-d5c9d9ebbf11">
<img width="240" alt="Screenshot 2023-12-05 at 2 55 15 PM" src="https://github.com/uchicago-cs/chigame/assets/121977204/87f4485e-1629-4a94-96c1-411cefe09b88">

Reply:
To add onto this, the deselecting when the other one is selected is not completely working. It works from like --> unlike but not the opposite way, but this can be a minor PR.